src/modules/ipops/doc/ipops_admin.xml

```diff
@@ -883,6 +883,57 @@ if(dns_query("test.com", "xyz"))
 
     </section>
 
+    <section id="ipops.f.ptr_query">
+      <title>
+        <function moreinfo="none">ptr_query(ip, pvid)</function>
+      </title>
+
+      <para>
+		  Store the hostname that correspond to ip
+		  in a config variable $ptrquery(pvid=>hostname).
+      </para>
+
+      <para>Parameters:</para>
+
+      <itemizedlist>
+        <listitem>
+          <para>
+			  <emphasis>ip</emphasis> - string or pseudo-variable containing the ip.
+			  The resulting IP addresses from DNS query are compared with ipaddr.
+          </para>
+        </listitem>
+        <listitem>
+          <para>
+            <emphasis>pvid</emphasis> - container id for script variable.
```

Comment:
I don't think we call a pseudo variable a "container" anywhere. If so, I consider it a bug :-)

Reply:
Actually the module uses already the term "container" in 3 places. So its probably fine in this regards.

Reply:
These were mostly taken from dns_query docs and just altered it where it made sense.

Reply:
What is the correct term for the `member/container` inside a pv that has multiple keys to get info from?